### PR TITLE
Retab publish distro

### PR DESCRIPTION
**Pull request by lkocman**
* based on https://github.com/openSUSE/openSUSE-release-tools/pull/2903#discussion_r1035751458
* depends on https://github.com/openSUSE/openSUSE-release-tools/pull/2903
* retab with two spaces insetad of tabs
* supply vim modeline

### CODE REVIEW

**Review by mvidner — CHANGES_REQUESTED** [review]
I'd like to point out that https://github.com/openSUSE/openSUSE-release-tools/blob/master/.editorconfig exists and says that `*` files (shell) use 2 spaces
Your choice of 4 spaces is inconsistent :)

**Comment by lkocman** [comment]
> I'd like to point out that https://github.com/openSUSE/openSUSE-release-tools/blob/master/.editorconfig exists and says that `*` files (shell) use 2 spaces Your choice of 4 spaces is inconsistent :)

Force pushed with two tabs instead

File: publish_distro [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+# vim:sw=4:ts=4:et:
```
Comment (dirkmueller) [inline publish_distro:3]: That now needs updating or removal

**Review by dirkmueller — CHANGES_REQUESTED** [review]
See inline comment

**Comment by coolo** [comment]
Superseded by #2910 (fixing the editor config conformance for all)

**Pull request closed without merging**